### DISCUSSION TITLE
Help: sunshine cannot be connected when used with zerotier 求助：sunshine配合zerotier使用无法连接

### DISCUSSION THREAD

**Discussion by Xemember** [LizardByte/community#365] [Sunshine]
sunshine配合zerotier使用时.未监听zerotier的虚拟网卡对应端口.无法连接到主机.何解？
（后补截图位）

**Comment by ReenigneArcher** [comment]
Use english

**Reply by Xemember** [reply]
Sorry. The following is using Google Translate 

When sunshine is used with zerotier, the port corresponding to zerotier's virtual network card is not monitored and cannot be connected to the host.

[a comment by an automated account is not shown]
